scripts/ci/run_csi_test_client.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash -xe
 
-[ $# -ne 2 ] && { echo "Usage $0 : container_name folder_for_junit"  ; exit 1; }
+[ $# -ne 3 ] && { echo "Usage $0 : container_name folder_for_junit"  ; exit 1; }
 
 #/tmp/k8s_dir is the directory of the csi grpc\unix socket that shared between csi server and csi-test docker 
-docker build -f Dockerfile-csi-test -t csi-sanity-test .  && docker run --user=root -e STORAGE_ARRAYS=${STORAGE_ARRAYS} -e USERNAME=${USERNAME} -e PASSWORD=${PASSWORD}  -e POOL_NAME=${POOL_NAME} -v /tmp/k8s_dir:/tmp/k8s_dir:rw  -v$2:/tmp/test_results:rw --rm  --name $1 csi-sanity-test
+if [ $3 = 'community_svc' ] ; then
+    docker build -f Dockerfile-csi-svc-test -t csi-sanity-test .  && docker run --user=root -e STORAGE_ARRAYS=${STORAGE_ARRAYS} -e USERNAME=${USERNAME} -e PASSWORD=${PASSWORD}  -e POOL_NAME=${POOL_NAME} -v /tmp/k8s_dir:/tmp/k8s_dir:rw  -v$2:/tmp/test_results:rw --rm  --name $1 csi-sanity-test
```

Comment:
instead of building from the separate `Dockerfile-csi-svc-test`, simply pass a different `PARAM_FILE` for `docker run` here